service/src/main/kotlin/fi/espoo/evaka/titania/TitaniaController.kt

```diff
@@ -35,6 +35,13 @@ class TitaniaController(private val titaniaService: TitaniaService) {
             lateinit var result: UpdateWorkingTimeEventsServiceResponse
             dbc.transaction { tx -> result = titaniaService.updateWorkingTimeEvents(tx, request) }
             result.createdEmployees.forEach { Audit.EmployeeCreate.log(targetId = AuditId(it)) }
+            if (!result.overlappingShifts.isEmpty())
```

Comment:
`foo.isNotEmpty()` might be a bit more readable than `!foo.isEmpty()`. Also needs curly braces